website/payments/signals.py

```diff
@@ -21,3 +21,32 @@ def give_new_users_tpay_permissions(instance, created, **kwargs):
         # after this signal call (triggered by the model's `save()`), a
         # `save_related()` will be called which will override the permissions
         # again.
+
+
+@suspendingreceiver(post_save, sender="payments.BankAccount")
+def clear_enabled_cache(sender, instance, **kwargs):
+    cache = PaymentUserCache.objects.filter(user=instance.owner).first()
+    if cache is not None:
+        cache.enabled = None
+        cache.save()
+
+
+@suspendingreceiver(post_save, sender="auth.Permission")
```

Comment:
This one is not triggered when I use the admin action to allow tpay for a user (which is the default way to allow tpay for a user)

Reply:
that admin just changes the permission right?

Reply:
The admin action calls this: https://github.com/svthalia/concrexit/blob/d2741f10fb6395e58862aaa888ee3c30d2c1b1fe/website/payments/admin.py#L715-L724

Reply:
I'd say that is specifically written to not do a full update on a queryset. So this is weird.